alacritty/src/logging.rs

```diff
@@ -149,6 +149,14 @@ fn create_log_message(record: &log::Record<'_>, target: &str) -> String {
     message
 }
 
+/// Check if log messages from a crate should be logged.
+fn is_allowed_target(level: Level, target: &str) -> bool {
+    match (level, log::max_level()) {
+        (Level::Error, LevelFilter::Trace) | (Level::Warn, LevelFilter::Trace) => true,
```

Comment:
I think on the latest Rust version it should be possible to do `Level::Warn | Level::Error` inside the tuple. I'm not sure when that was introduced, but our CI should be able to verify that.

Reply:
I have made this change and it compiles on my machine (Rust version 1.54.0). I will wait until this passes CI to resolve this conversation.

Reply:
It's newer than 1.50.0, so we can't use this unfortunately. I've fixed it myself.